src/tools/cephfs/top/cephfs-top

```diff
@@ -452,18 +450,21 @@ class FSTop(object):
                         remaining_hlen = 0
                     else:
                         remaining_hlen -= coord[1]
+                    size = 0
                     if key == "READ_IO_SIZES":
-                        global last_read_size
-                        last_size = last_read_size.get(client_id, 0)
-                        size = m[1] - last_size
-                        last_read_size[client_id] = m[1]
+                        if m[1] > 0:
```

Comment:
```suggestion
                       if key == "READ_IO_SIZES" and m[1] > 0:
```
Maybe this way it looks a bit better?

Reply:
I'd like to keep it this way, because it's easier to analyze 'key' values later in refresh_client(). There are many values to refresh, so keeping them similar.